qiskit/transpiler/passes/unroller.py

```diff
@@ -16,14 +17,13 @@ class Unroller(TransformationPass):
     to a desired basis, using decomposition rules defined for each instruction.
     """
 
-    def __init__(self, basis=None):
+    def __init__(self, basis):
         """
         Args:
             basis (list[qiskit.circuit.gate.Gate]): Target basis gates to unroll to.
```

Comment:
Unrelated with your PR, but it seems that there is a mistake in the docstring:
```suggestion
            basis (list[str]): Target basis gate names to unroll to, e.g. `['U', 'CX']` .
```

Reply:
The issue's object was to make the basis explicit. I think making it a required arg instead of default to `None` is appropriate. Why do you suggest that be the default?

Reply:
I agree with removing the default. My comment was about the type of the parameter in the docstring. The rest in an example.